src/runtime/virtcontainers/network_linux.go

```diff
@@ -597,7 +597,7 @@ func xConnectVMNetwork(ctx context.Context, endpoint Endpoint, h Hypervisor) err
 	queues := 0
 	caps := h.Capabilities(ctx)
 	if caps.IsMultiQueueSupported() {
-		queues = int(h.HypervisorConfig().NumVCPUs())
+		queues = int(h.HypervisorConfig().NetworkQueues)
```

Comment:
Here, the original queues which comes from `int(h.HypervisorConfig().NumVCPUs())` is equal to the current vcpus. 
If we want to check the validation of network queues setting , maybe it is a choice, or there's a better one.